### DISCUSSION TITLE
what are the differences from GPT-engineer?

### DISCUSSION THREAD

**Discussion by ft-fujii** [Pythagora-io/gpt-pilot#202] [Q&A]
It looks like the both have the same concept.
What are the differences?

**Comment by nalbion** [comment]
As far as I can tell, GPT Engineer has a single persona which can generate a new project or make improvements to an existing project, and - and then it's done.

GPT Pilot is modelled on a real-world software development team, with a number of agents with specialist skills. It's early days yet, but I believe this model allows for a lot more versatility and extensibility in the future.

**Reply by ft-fujii** [reply]
interesting!

**Comment by TerminatedProcess** [comment]
Will it work with existing projects?
